lib/cassandra_record/base.rb

```diff
@@ -20,6 +20,24 @@ def batch_create(array_of_attributes, options={})
         array_of_attributes.map { |attr| new(attr) }
       end
 
+      # combines results from all pages
+      # into a single result set
+      def all(attributes={})
+        paginated_result_set = where(attributes)
+        rows = []
+
+        loop do
+          paginated_result_set.each do |row|
```

Comment:
The underlying gem returns an Enumerable `Result`, but that `Result` is actually paginated, thus the bug with our implementation.

My intention here is to leave the `.where` method as is to allow clients to paginate however they'd like, but add the `.all` method for clients that would prefer to treat the results as a complete, unpaginated set.